packages/react-components/src/SummaryBox.tsx

```diff
@@ -22,14 +22,19 @@ export default React.memo(styled(SummaryBox)`
   align-items: stretch;
   border-radius: 0.25rem;
   display: flex;
-  flex-wrap: no-wrap;
-  justify-content: space-between;
+  flex-wrap: nowrap;
```

Comment:
This is the first thing that caught my eye - certainly not a direction I believe we should go in, it looks all squashed into the left side now.

Reply:
I've restored the `space-betweend` ✅